shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/APIToken.kt

```diff
@@ -0,0 +1,28 @@
+package co.nimblehq.blisskmmic.domain.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class APIToken(
```

Comment:
We have this convention [on Compass](https://nimblehq.co/compass/development/code-conventions/kotlin/#properties): Acronyms should be treated as words

```suggestion
data class ApiToken(
```

Welcome to Kotlin world! 🙌